.goreleaser.yaml

```diff
@@ -96,8 +96,8 @@ dockers:
       - --platform=linux/amd64
     image_templates:
       - "flashbots/suave-geth:{{ .ShortCommit }}-amd64"
-      - "flashbots/suave-geth:{{ .Tag }}-amd64"
-      - "flashbots/suave-geth:latest-amd64"
+      - "flashbots/suave-geth:v{{ .Version }}-amd64"
```

Comment:
It was using Tag before, but it needs to use Version because that is the field that the `nightly` section overrides during nightly releases.